e2e/etcd_test.go

```diff
@@ -443,6 +443,18 @@ func spawnWithExpects(args []string, xs ...string) error {
 	return perr
 }
 
+func spawnWithExpectNoOutput(args []string) error {
+	proc, err := spawnCmd(args)
+	if err != nil {
+		return err
+	}
+	rs, err := proc.ExpectFunc(func(txt string) bool { return txt == "" })
```

Comment:
Why not just count the number of received lines? I think the matching on "" won't work as expected since etcdctl could print lines which won't match "" so it'll pass as `len(rs) == 0`.

Line counting seems simpler:
```go
if err := proc.Close(); err != nil { return err }
if proc.LineCount() != 0 { return fmt.Errorf("unexpected output") }
```